### PR TITLE
CVSB-840 - Add test code

### PR DESCRIPTION
**Pull request by acozianu**

### CODE REVIEW

File: src/models/TestResultsSchema.js [hunk 1]
```diff
@@ -65,25 +65,26 @@ const testResultsSchema = Joi.object().keys({
...
   testEndTimestamp: Joi.date().iso().required(),
-  testStatus: Joi.any().only([ 'Submitted', 'Canceled' ]).required(),
+  testStatus: Joi.any().only([ 'submitted', 'canceled' ]).required(),
   reasonForCancellation: Joi.string().max(500).required(),
-  vehicleClass: Joi.any().only(['2', 'S', '1', 'T', 'L', '3', 'V']).required(),
-  vehicleType: Joi.any().only(['psv', 'hgv', 'trl', 'others']).required(),
-  numberOfSeats: Joi.number().min(1).required(),
+  vehicleClass: Joi.any().only(['2', 's', '1', 't', 'l', '3', 'v']).required(),
+  vehicleType: Joi.any().only(['psv', 'hgv', 'trl']).required(),
+  numberOfSeats: Joi.number().min(1).max(4).required(),
```
Comment (gmlazu) [inline src/models/TestResultsSchema.js]: There should not be any maximum number of seats

File: src/models/TestResultsSchema.js [hunk 2]
```diff
@@ -65,25 +65,26 @@ const testResultsSchema = Joi.object().keys({
...
   vehicleStatus: Joi.any().only(['1', '2', '3']).required(),
   vehicleConfiguration: Joi.any().only(['rigid', 'articulated']).required(),
-  odometerReading: Joi.number().required(),
+  odometerReading: Joi.number().min(1).max(7).required(),
```
Comment (gmlazu) [inline src/models/TestResultsSchema.js]: Odometer readings can range from 0 to infinity

Comment (oalexandru) [reply src/models/TestResultsSchema.js]: fixed

File: src/models/TestResultsSchema.js (hunk 1 above)
Comment (oalexandru) [reply src/models/TestResultsSchema.js]: fixed

**Pull request merged**